app/_how-tos/send-asychronous-llm-requests-to-save-costs.md

```diff
@@ -0,0 +1,289 @@
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg
+---
+
+## Configure AI Proxy plugins
+
+Configure two separate AI Proxy plugins: one for the `llm/v1/files` route and another for the `llm/v1/batches` route. Each route type requires its own dedicated service and route to function correctly. In this setup, all requests to the files route are forwarded to `/files` endpoint, while batch requests go to `/batches` endpoint.
+
+
+AI Proxy plugin for the `route_type: llm/v1/files` :
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: ai-proxy
+      service: files-service
+      config:
+        model_name_header: false
+        route_type: llm/v1/files
+        auth:
+          header_name: Authorization
+          header_value: Bearer ${openai_api_key}
+        model:
+          provider: openai
+variables:
+  openai_api_key:
+    value: $OPENAI_API_KEY
+{% endentity_examples %}
+
+AI Proxy plugin for the `route_type: llm/v1/batches`:
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: ai-proxy
+      service: batches-service
+      config:
+        model_name_header: false
+        route_type: llm/v1/batches
+        auth:
+          header_name: Authorization
+          header_value: Bearer ${openai_api_key}
+        model:
+          provider: openai
+variables:
+  openai_api_key:
+    value: $OPENAI_API_KEY
+{% endentity_examples %}
+
+# Upload a .jsonl file for batching
+
+Use the following command to upload your [batching file](./#batch-jsonl-file) to the `/files` route:
+
+```bash
+curl localhost:8000/files -F purpose="batch" -F file="@batch.jsonl"
+```
+
+If the upload succeeds, you will see a JSON response like this:
+
+```json
+{
+  "object": "file",
+  "id": "file-abc123xyz456789lmn0pq",
+  "purpose": "batch",
+  "filename": "1.jsonl",
+  "bytes": 1672,
+  "created_at": 1751281528,
+  "expires_at": null,
+  "status": "processed",
+  "status_details": null
+}
+```
+
+{:.success}
+> **Note:** Copy the file ID from the response, you will need it to create a batch. Export it as an environment variable:
+>
+> ```bash
+>   export FILE_ID=YOUR_FILE_ID
+> ```
+
+## Create a batching request
+
+Send a POST request to the `/batches` route to create a batch using your uploaded file. Make sure to replace `YOUR_FILE_ID` with the ID of your uploaded file from the previous step:
+
+{:.info}
+> The completion window must be set to `24h`, as it's the only value currently supported by the OpenaAI /batches API
+>
+> In this example we use the `/v1/chat/completions` route for batching because we are sending multiple structured chat-style prompts in OpenAI's chat completions format to be processed in bulk.
+
+```bash
+curl http://localhost:8000/batches \
+  -H "Content-Type: application/json" \
+  -d "{
+    \"input_file_id\": \"$FILE_ID\",
+    \"endpoint\": \"/v1/chat/completions\",
+    \"completion_window\": \"24h\"
+  }"
+```
```

Comment:
Even though it's not technically a validation step, I think you could use a `{% validation request-check %}` block for this (but maybe double-check with Fabian)